src/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -15,11 +15,14 @@ using LinearAlgebra.BLAS: gemv, gemv!, gemm!, trsm!, axpy!, ger!
 # for derivations for wide and tall matrices, see
 # https://sethaxen.com/blog/2021/02/differentiating-the-lu-decomposition/
 
+const _RowMaximum = VERSION >= v"1.7.0-DEV.1188" ? RowMaximum : Val{true}
+const _NoPivot = VERSION >= v"1.7.0-DEV.1188" ? NoPivot : Val{false}
```

Comment:
Do we need to call this LU_NoPivot
since we will want this to differ for QR?

Reply:
Yeah, makes sense